src/PropertyDamageRisk.src.json

```diff
@@ -49,5 +49,32 @@
             }
         },
         "context": {}
+    }, {
+        "title": "A PropertyDamageRisk for a building",
+        "tab": "tab-expanded",
+        "input": {
+            "@context": [
+                "http://schema4i.org/Thing.jsonld",
+                "http://schema4i.org/Building.jsonld",
+                "http://schema4i.org/Risk.jsonld",
+                "http://schema4i.org/PropertyDamageRisk.jsonld"
+            ],
+            "@type": "PropertyDamageRisk",
+            "Affects": {
+                "@type": "Building",
+                "DamageCause": "F",
```

Comment:
`DamageCause` is an attribute of `PropertyDamageRisk`. Please correct the sample.